.circleci/config.yml

```diff
@@ -261,12 +268,21 @@ jobs:
           username: dockstoretestuser
           password: $DOCKERHUB_PASSWORD
     steps:
-      - get_workspace
-      # Override the webservice version in package.json to instead build against develop webservice
-      - run:
-          name: set webservice to develop
-          command: bash -i -c 'npm config set dockstore-ui2:webservice_version develop'
-      - build_ui
+      - run: # should not need this, but circle ci does not like it when a condition wipes out a whole job https://discuss.circleci.com/t/pipeline-parameters-in-steps-when-conditions-seems-to-have-no-value-using-dynamic-configuration/50524/2
+          name: Java/Maven/Python versions
+          command: |
+            java -version
+            npm --version
+      - when:
+          condition:
+            not: << pipeline.parameters.sanity-check-against-develop >>      
```

Comment:
I'm a little confused by the purpose of these steps if the sanity check against develop is also happening in `lint_license_unit_test_coverage` above. Could you elaborate? 

Also should that `not` be there?

Reply:
The basic idea is that it occurs in one place or the other. 
It  used to occur here on, but only on the develop branch and it would only check that the build passed (`build_ui1`) with no tests being executed. When `sanity-check-against-develop` is false, nothing really changes from before. 

When `sanity-check-against-develop` is true, this step basically disappears and moves to `lint_license_unit_test_coverage` so that, hopefully, the integration tests run against develop since the workspace is being attached (which should carryover https://support.circleci.com/hc/en-us/articles/10816400480411-How-to-pass-environment-variables-between-jobs )